debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlEnumColumnIT.java

```diff
@@ -70,8 +70,8 @@ public void shouldAlterEnumColumnCharacterSet() throws Exception {
         String allowedBeforeAlter = schemaBeforeAlter.field("type").schema().parameters().get("allowed");
         String allowedAfterAlter = schemaAfterAlter.field("type").schema().parameters().get("allowed");
 
-        assertThat(allowedBeforeAlter).isEqualTo("station,post_office");
-        assertThat(allowedAfterAlter).isEqualTo("station,post_office,plane,ahihi_dongok,now");
+        assertThat(allowedBeforeAlter).isEqualTo("['station','post_office']");
```

Comment:
See my other comment below, the new brackets are potentially breaking existing downstream users of the message schema.